src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/prison/PrisonApiClientTest.kt

```diff
@@ -0,0 +1,69 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.prison
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import com.github.tomakehurst.wiremock.client.WireMock.aResponse
+import com.github.tomakehurst.wiremock.client.WireMock.post
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig
+import com.github.tomakehurst.wiremock.junit5.WireMockExtension
+import com.github.tomakehurst.wiremock.stubbing.Scenario.STARTED
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.RegisterExtension
+import org.springframework.web.reactive.function.client.WebClient
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.TestData.hdcPrisonerStatus
+
+class PrisonApiClientTest {
+
+  companion object {
+    @RegisterExtension
+    val wiremock = WireMockExtension.newInstance().options(wireMockConfig().dynamicPort()).build()
```

Comment:
These are new - integration testing of API clients independently of service or integration tests.